README.md

```diff
@@ -276,6 +276,8 @@ The Hash of EC tag name/value pairs which will be applied to the instance.
 
 The default is `{ "created-by" => "test-kitchen" }`.
 
+Tags are applied post creation for spot instances and during creation for standard instances.
```

Comment:
```suggestion
```

We can just get rid of this note now since all tags are added during creation, correct?